charts/external-metrics/values.yaml

```diff
@@ -11,9 +11,8 @@ prometheus:
       size: 10Gi
   kube-state-metrics:
     enabled: true
-    # TODO (kyuds): remove skypilot-cluster label in v0.11.0; deprecated in favor of skypilot-cluster-name.
```

Comment:
@kyuds  - this change was made fairly recently. Echoing @SeungjinYang's [comment](https://github.com/skypilot-org/skypilot/pull/7724#issuecomment-3483120389), I'm a bit concerned about deprecating it in v0.11.0 (e.g., old cluster and user upgrades API server/installs prometheus from new chart).

Do you think this deprecation should be moved to v0.12.0 like others in #7724?